Task 3/Knapsack/knapsack.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
```

Comment:
Can we have a small explanation & sample Input/Output cases as a part of a multiline comment at the top here?

Reply:
Could you please add a link to this question? 
It explains the question along with the example. 
https://leetcode.com/problems/sort-colors/

Reply:
Alright @AnkitaxPriya.

Go ahead and add that link as a part of a comment on the top of the program!